backend/src/services/v2/release.ts

```diff
@@ -106,6 +116,30 @@ export async function createRelease(user: UserDoc, releaseParams: CreateReleaseP
   return release
 }
 
+export type UpdateReleaseParams = Pick<ReleaseInterface, 'notes' | 'draft' | 'fileIds' | 'images'>
+export async function updateRelease(user: UserDoc, modelId: string, semver: string, delta: UpdateReleaseParams) {
```

Comment:
Do we need to reset approvals for a release? Do we need to send emails notifying people that a review has been updated?